config/webpack.client.js

```diff
@@ -62,6 +62,7 @@ module.exports = (env, argv) => {
       new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
       isProduction && new DuplicatePackageCheckerPlugin(),
       new webpack.IgnorePlugin(/^jsdom$/),
+      new webpack.IgnorePlugin(/unicode\/category\/So/),
```

Comment:
This 2mb file with unicode chars can safely be ignored: https://yarnpkg.com/en/package/slug